examples/main-empty/index.ts

```diff
@@ -1,8 +1,15 @@
 import { serve } from "https://deno.land/std@0.131.0/http/server.ts";
 import {sum} from "./some-import.ts";
+import isEven from "npm:is-even";
+
+console.log('Hello A');
+globalThis.isTenEven = isEven(10);
 
 console.log(Deno.version);
 let val = sum(1, 2);
+console.log(Deno.readFile)
+console.log(Deno.cwd())
+console.log(Deno.readFileSync('examples/postgres-on-the-edge/README.md'));
```

Comment:
In your recent changes, you seem to have decided to prefix `./mnt/data`
I think this path should be changed to reflect that change as well 😎